parser/digester.go

```diff
@@ -231,6 +269,65 @@ func (d *sqlDigester) normalize(sql string, keepHint bool) {
 	d.tokens.reset()
 }
 
+// Normalize the sql command with additional binding specific rules
+func (d *sqlDigester) normalizeForBinding(sql string, keepHint bool) {
```

Comment:
It's very similar with `sqlDigester.normalize`, how about merging them into one?
For example:
```
func (d *sqlDigester) normalize(sql string, keepHint, forBinding bool) {
    ...
    if forBinding {d.reduceInListWithSingleLiteral(&currTok)}
    ...
}
```